Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.py

```diff
@@ -1,15 +1,23 @@
 from CommonServerPython import *
+import demistomock as demisto
+from GSuiteApiModule import *  # noqa: E402
```

Comment:
restore the location of the import

Reply:
https://github.com/demisto/content/commit/77e493dff369e8aa5d500f61d36769941541f40e

Reply:
https://github.com/demisto/content/commit/e73f0309c72de8052a37c1437d76b19697b9ea4e